src/components/questions/addAnswer.js

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import axios from "axios";
+
+class AddAnswer extends React.Component {
```

Comment:
Is it possible you could reuse these components and just have some conditional rendering for answers and questions? Or maybe make a higher order component to manage state and pass in what is displayed like we did with Modal.